src/test/java/hudson/plugins/warnings/parser/MetrowerksCWCompilerParserTest.java

```diff
@@ -0,0 +1,79 @@
+package hudson.plugins.warnings.parser;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Iterator;
+
+import org.junit.Test;
+
+import hudson.plugins.analysis.util.model.FileAnnotation;
+import hudson.plugins.analysis.util.model.Priority;
+
+/**
+ * Tests the class {@link MetrowerksCWCompilerParser}.
+ */
+public class MetrowerksCWCompilerParserTest extends ParserTester {
+    /** Error message. */
+    private static final String WRONG_NUMBER_OF_WARNINGS_DETECTED = "Wrong number of warnings detected.";
+    /** An info. */
+    private static final String INFO_CATEGORY = "Info";
+    /** A warning. */
+    private static final String WARNING_CATEGORY = "Warning";
+    /** An error. */
```

Comment:
WARNING_CATEGORY is needed below. Otherwise I have to use the string directly. Is there a better solution?